ui/DesignSystem/Component/Header/Back.svelte

```diff
@@ -0,0 +1,44 @@
+<script>
+  import { pop } from "svelte-spa-router";
```

Comment:
Components shouldn't touch routing, instead we should just `dispatch` an event when the back buton is clicked. With that the screen can react to that event and decide what suppose to happen.

Reply:
Yeah I thought about that but the back button will always just pop. So I think it makes sense to contain it within that. Otherwise we need to duplicate that on all the pages you use it and that will lead to it breaking.